ledger-tool/src/main.rs

```diff
@@ -1150,7 +1150,10 @@ fn main() {
         .long("accounts-hash-cache-path")
         .value_name("PATH")
         .takes_value(true)
-        .help("Use PATH as accounts hash cache location");
+        .help("Use PATH as accounts hash cache location")
+        .long_help(
```

Comment:
why not in `.help()`?  afaik, clap would automatically append a string to `.help()` for `.default()`'s value if we could use it in this case.